http/integration/integration_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2024 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package integration_test
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/http/httputil"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/urfave/negroni/v3"
```

Comment:
I would move this to folder `http/integration/negroni` since I guess the main target of the test is negroni